tt_metal/impl/dispatch/dispatch_core_manager.hpp

```diff
@@ -19,6 +19,45 @@ namespace tt::tt_metal {
 // One core dispatches commands to worker cores on the device `dispatcher`
 // The `remote_x` cores are used for remote fast dispatch and receive / transmit fast dispatch packets from ethernet cores
 
+enum DispatchWorkerType : uint32_t {
+    PREFETCH = 0,
+    PREFETCH_D = 1,
+    DISPATCH = 2,
+    DISPATCH_D = 3,
+    MUX = 4,
+    MUX_D = 5,
+    DEMUX = 6,
+    DEMUX_D = 7,
+    US_TUNNELER_LOCAL = 8,
+    US_TUNNELER_REMOTE = 9,
+    DS_TUNNELER_LOCAL = 10,
+    DS_TUNNELER_REMOTE = 11,
+    COUNT = 12
+};
+
+struct worker_build_settings_t{
```

Comment:
quibble: not a fan of this name.  dispatch_worker_build_settings_t ?  or dispatch_settings_t?  or cq?